### ISSUE TITLE
Some confusion regarding the implementation of virtual batch norm.

### ISSUE THREAD

**Issue by tumble-weed** [torchgan/torchgan#146]
**Describe the bug**
Some confusion regarding the implementation of virtual batch norm. 

1. Referring to another code: https://github.com/dansuh17/segan-pytorch/blob/938b6c837b6f091d0ad1e67fad9db32045a90151/vbnorm.py#L68 , it seems that in the torchgan version the mean and deviation are set to None after a batch has passed. in the next batch these will be computed for that sample and applied, and not from a "virtual batch". https://github.com/torchgan/torchgan/blob/f4139537ac2d3d8609d5aecc859a6fb797b107a1/torchgan/layers/virtualbatchnorm.py#L93 . so it seems for an alternate batch it uses statistics from that batch, and for the rest from the previous batch. 
2. even when the statistics are updated they are directly copied from a batch as opposed to interpolated as in https://github.com/dansuh17/segan-pytorch/blob/938b6c837b6f091d0ad1e67fad9db32045a90151/vbnorm.py#L66
